hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/Route.java

```diff
@@ -38,10 +40,12 @@ public void configure() {
 
         from("jetty:http://{{hostname}}:{{port}}/{{endpoint}}").routeId("hnsecure-route")
             .log("HNSecure received a request")
+            //.process(validator)
             .process(validateAccessToken).id("ValidateAccessToken")
             .setBody().method(new FhirPayloadExtractor())
             .log("Decoded V2: ${body}")            
-            // TODO if Payload validator is called beforeFhirPayloadExtractor(), no need of separate validator in validateAccessToken 
+            // TODO ADDRESSED if Payload validator is called beforeFhirPayloadExtractor(), no need of separate validator in validateAccessToken
```

Comment:
The TODO ADDRESSED can be removed before Merge. The comment portion can be left in if still relevant.